src/components/Navbar/navbar.scss

```diff
@@ -0,0 +1,110 @@
+.navbar {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 5rem;
+    display: flex;
+    align-items: center;
+    z-index: 200;
+    background: var(--clr-white);
+  }
+
+  .nav-center {
+    width: 90vw;
+    max-width: 1170px;
+    margin: 0 auto;
+  }
+  .nav-header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  .nav-header img {
+    margin-bottom: 0.375rem;
+  }
+
+.toggle-btn {
+  font-size: 1.6rem;
+  background: transparent;
+  border-color: transparent;
+  color: var(--clr-primary-5);
+  cursor: pointer;
+  transition: var(--transition);
+}
+.toggle-btn:hover {
+  color: #902413;
+}
+.nav-links {
+  display: none;
+}
+
+.show-nav {
+  height: 500px;
+}
+  @media screen and (min-width: 992px) {
```

Comment:
It will be a good practice to format the scss files. There are many formatted available in vscode.